db.go

```diff
@@ -27,6 +30,11 @@ type database interface {
 	GetNotifications(to int64) ([]*ScheduledNotification, error)
 	GetMetricsCount() (int64, error)
 	GetChecksCount() (int64, error)
+	GetUsernameID(login string) (string, error)
```

Comment:
A bad name for two reasons:

1. It's actually get-id-by-username, not get-username-by-id as the name suggests. It takes username and returns an id. So better call it `GetIDByUsername`.
2. This name is too generic. It is in the same namespace with all other database functions, and it does not specify that it relates only to Twitter usernames and ids. So better call it `GetTwitterIDByUsername` or something.